src/components/AeAmount.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <span class="ae-amount">
+      <span>{{ roundedAmount }}</span>
+    </span>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
```

Comment:
why have unused import? it would be good if you use a IDE/Editor that can help with these things